modules/arm_plugin/src/transformations/arm_optimizations.cpp

```diff
@@ -242,7 +243,7 @@ bool ArmPlugin::pass::ArmOptimizations::run_on_function(std::shared_ptr<ngraph::
 
     {
         Dump(f, "before_arm_specific_transformations");
-        ngraph::pass::Manager manager;
+        ov::pass::Manager manager;
```

Comment:
Lines 248-251, 261-262,265 and probably some more are local `MatcherPass`-es also